app/views/products/edit.html.erb

```diff
@@ -0,0 +1,42 @@
+<%= form_with model: @product, url: product_path, method: :patch do |f| %>
```

Comment:
Why we're not creating a form partial and move both new and edit product forms on it?